### PR TITLE
DMU: Implement 4 cards

### PR DESCRIPTION
**Pull request by ZacharyDeganutti**
- Aether Channeler
- Cult Conscript
- Phoenix Chick
- Urborg Lhurgoyf

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/phoenix_chick.txt [hunk 1]
```diff
@@ -0,0 +1,15 @@
+Name:Phoenix Chick
+ManaCost:R
+Types:Creature Phoenix
+PT:1/1
+K:Flying
+K:Haste
+K:CARDNAME can't block.
+T:Mode$ AttackersDeclared | CheckSVar$ X | SVarCompare$ GE3 | Execute$ TrigReturn | NoResolvingCheck$ True | TriggerZones$ Graveyard | AttackingPlayer$ You | TriggerDescription$ Whenever you attack with three or more creatures, you may pay {R}{R}. If you do, return Warcry Phoenix from your graveyard to the battlefield tapped and attacking with a +1/+1 counter on it.
+SVar:TrigReturn:AB$ ChangeZone | Cost$ R R | Defined$ Self | Origin$ Graveyard | Destination$ Battlefield | Tapped$ True | Attacking$ True | SubAbility$ AddCounter
```
Comment (Hanmac) [inline forge-gui/res/cardsfolder/upcoming/phoenix_chick.txt:9]: should be `WithCounterType$ P1P1`

Comment (ZacharyDeganutti) [reply forge-gui/res/cardsfolder/upcoming/phoenix_chick.txt:9]: I'm a bit new to this, did a grep through the codebase and didn't find any references to "WithCounterType", and replacing the SubAbility portions with "WithCounterType$ P1P1" didn't seem to have the intended effect. Can you elaborate on what this means?

Comment (Hanmac) [reply forge-gui/res/cardsfolder/upcoming/phoenix_chick.txt:9]: ah typo its `WithCountersType$ P1P1`

Comment (ZacharyDeganutti) [reply forge-gui/res/cardsfolder/upcoming/phoenix_chick.txt]: Should be good to go now.

**Review by tool4ever — APPROVED** [review]

**Pull request merged**